packages/gasket-intl/test/components.test.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import assume from 'assume';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+import { mount } from 'enzyme';
+import { IntlProvider } from 'react-intl';
+import { ERROR, LOADED } from '../src/utils';
+
+const fetchStub = sinon.stub();
+const mockManifest = require('./fixtures/mock-manifest.json');
+const mockConfig = {};
```

Comment:
You're modifying this in tests (at least in the beforeEach, you should move the declaration into the `describe`, change it to a `let` and just make a new mockConfig for each test.